editor/src/clj/editor/gui.clj

```diff
@@ -2349,6 +2365,8 @@
   (input textures-node g/NodeID) ; for tests
   (input particlefx-resources-node g/NodeID) ; for tests
   (input handler-infos g/Any :array)
+  (output handler-infos g/Any (g/fnk [handler-infos]
+                                (into [] (mapcat one-or-many-handler-infos-to-vec) handler-infos)))
```

Comment:
I believe the `handler-infos` input is used by `defold/extension-spine` here, if you're wondering why it was added: https://github.com/defold/extension-spine/blob/main/defold-spine/editor/src/spineguiext.clj#L341